docs/basics/ExampleTodoList.md

```diff
@@ -190,7 +191,8 @@ export default connect(select)(App);
 #### `components/AddTodo.js`
 
 ```js
-import React, { findDOMNode, Component, PropTypes } from 'react';
+import React, { Component, PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
```

Comment:
Instead of adding `findDOMNode` we can instead change the example to use the ref directly:

```js
const node = findDOMNode(this.refs.input);
```

becomes

```js
const node = this.refs.input;
```